guide/additional-info/changes-in-v14.md

```diff
@@ -539,22 +545,36 @@ Added `Guild#setMFALevel()` which sets the guild's MFA level.
 
 `videoQualityMode` may be used whilst creating a channel to initially set the camera video quality mode.
 
+### GuildEmojiManager
+
+Added `GuildEmojiManager#delete()` and `GuildEmojiManager#edit()` for managing existing guild emojis.
+
+### GuildForumThreadManager 
+
+Added `GuildForumThreadManager` as manager for threads in forum channels as of 14.4.0.
+
 ### GuildMemberManager
 
 Added `GuildMemberManager#fetchMe()` to fetch the client user in the guild.
 
-### GuildEmojiManager
+### GuildTextThreadManager
 
-Added `GuildEmojiManager#delete()` and `GuildEmojiManager#edit()` for managing existing guild emojis.
+Added `GuildTextThreadManager` as manager for threads in text channels and announcement channels as of 14.4.0.
 
 ### Interaction
 
 Added `Interaction#isRepliable()` to check whether a given interaction can be replied to.
 
+### Message
+
+`Message#position` has been added as an approximate position in a thread as of 14.4.0.
+
 ### MessageReaction
 
 Added `MessageReaction#react()` to make the client user react with the reaction the class belongs to.
 
 ### Webhook
 
 Added `Webhook#applicationId`.
+
+Added `Webhook#threadName` as of 14.4.0 which allows a webhook to create a post in a forum channel.
```

Comment:
Not on Webhook, but when sending a webhook in the options.

Reply:
Ah yeah

Reply:
Well, mention that a new option exists / demonstrate it with a minimal use sample